indico/modules/events/papers/models/call_for_papers.py

```diff
@@ -69,3 +71,41 @@ def set_reviewing_state(self, reviewing_type, enable):
             paper_reviewing_settings.set(self.event, 'content_reviewing_enabled', enable)
         elif reviewing_type == 'layout':
             paper_reviewing_settings.set(self.event, 'layout_reviewing_enabled', enable)
+
+    @property
+    def managers(self):
+        return {p.principal for p in self.event.acl_entries
+                if p.has_management_role('paper_manager', explicit=True)}
```

Comment:
Hm, it's true, doesn't make sense since you cannot assign paper managers to papers... However, we do have it available in legacy. I guess this is one of the features we don't really want to migrate.